### PR TITLE
OCM-11127 | ci: fix id:73753,55883,75927

### PR DESCRIPTION
**Pull request by aaraj7**

### CODE REVIEW

File: tests/e2e/test_rosacli_cluster.go [hunk 1]
```diff
@@ -3530,6 +3530,7 @@ var _ = Describe("Sts cluster creation supplemental testing",
 				testOperatorRolePrefix := common.GenerateRandomName("opp75927", 2)
 				flags, err := profilehandler.GenerateClusterCreateFlags(customProfile, rosaClient)
 				Expect(err).ToNot(HaveOccurred())
+				rolePrefix = flags[16]
```
Comment (radtriste) [inline tests/e2e/test_rosacli_cluster.go:3533]: Is this arbitrary ? SHouldn't you use `GetFlagValue` ?

File: tests/e2e/test_rosacli_upgrade.go [hunk 2]
```diff
@@ -392,8 +392,6 @@ var _ = Describe("Cluster Upgrade testing",
 			By("Check upgrade state")
 			err = WaitForUpgradeToState(upgradeService, clusterID, con.Scheduled, 4)
 			Expect(err).To(BeNil())
-			err = WaitForUpgradeToState(upgradeService, clusterID, con.Started, 70)
```
Comment (radtriste) [inline tests/e2e/test_rosacli_upgrade.go:395]: We don't wait for upgrade started anymore ?

Comment (aaraj7) [reply tests/e2e/test_rosacli_upgrade.go:395]: If we will wait, the upgrade started will never be cancelled/deleted for next test cases.

File: tests/e2e/test_rosacli_cluster.go (hunk 1 above)
Comment (aaraj7) [reply tests/e2e/test_rosacli_cluster.go:3533]: The next test case uses same method, so I became just consistent, If you will tell I will change, wdyt?

File: tests/e2e/test_rosacli_upgrade.go (hunk 2 above)
Comment (radtriste) [reply tests/e2e/test_rosacli_upgrade.go:395]: In the test case I see `Check the upgrade is finished successfully`
So we should add a step to wait for the upgrade

File: tests/e2e/test_rosacli_cluster.go (hunk 1 above)
Comment (radtriste) [reply tests/e2e/test_rosacli_cluster.go:3533]: I would rather use some `discovery` than using hardcoded index which may change quite easily

File: tests/e2e/test_rosacli_upgrade.go (hunk 2 above)
Comment (aaraj7) [reply tests/e2e/test_rosacli_upgrade.go:395]: Okay will try to mould the test case same

File: tests/e2e/test_rosacli_cluster.go (hunk 1 above)
Comment (aaraj7) [reply tests/e2e/test_rosacli_cluster.go:3533]: Okay will change it Thanks

**Comment by radtriste** [comment]
@aaraj7 Could you rebase ?

File: tests/e2e/test_rosacli_upgrade.go [hunk 3]
```diff
@@ -969,6 +918,184 @@ var _ = Describe("ROSA HCP cluster upgrade",
...
+
+		var (
+			clusterID      string
+			clusterName    string
+			rosaClient     *rosacli.Client
+			clusterService rosacli.ClusterService
+			upgradeService rosacli.UpgradeService
+			versionService rosacli.VersionService
+			// ocmResourceService rosacli.OCMResourceService
+			// rolePrefix         string
```
Comment (radtriste) [inline tests/e2e/test_rosacli_upgrade.go:934]: remove if not used

File: tests/e2e/test_rosacli_upgrade.go [hunk 4]
```diff
@@ -969,6 +918,184 @@ var _ = Describe("ROSA HCP cluster upgrade",
...
+			profile *profilehandler.Profile
+		)
+
+		BeforeEach(func() {
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+			upgradeService = rosaClient.Upgrade
+			versionService = rosaClient.Version
+			// ocmResourceService = rosaClient.OCMResource
```
Comment (radtriste) [inline tests/e2e/test_rosacli_upgrade.go:944]: same here to be removed

File: tests/e2e/test_rosacli_upgrade.go [hunk 5]
```diff
@@ -969,6 +918,184 @@ var _ = Describe("ROSA HCP cluster upgrade",
 			})
 	})
 
+var _ = Describe("Sucessfull Upgrade Testing",
+	labels.Feature.Policy,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			clusterID      string
```
Comment (radtriste) [inline tests/e2e/test_rosacli_upgrade.go:927]: why do you make it shared ?

File: tests/e2e/test_rosacli_upgrade.go [hunk 6]
```diff
@@ -969,6 +918,184 @@ var _ = Describe("ROSA HCP cluster upgrade",
...
+				profile.NamePrefix = constants.DefaultNamePrefix
+
+				flags, err := profilehandler.GenerateClusterCreateFlags(profile, rosaClient)
+				Expect(err).ToNot(HaveOccurred())
+
+				clusterName = common.GenerateRandomName("cluster-55883", 2)
+				command := "rosa create cluster --cluster-name " + clusterName + " " + strings.Join(flags, " ") + " --mode auto"
+				rosalCommand := config.GenerateCommand(command)
+				log.Logger.Info(rosalCommand)
+				// rolePrefix = rosalCommand.GetFlagValue("--operator-roles-prefix", true)
```
Comment (radtriste) [inline tests/e2e/test_rosacli_upgrade.go:973]: to be removed

File: tests/e2e/test_rosacli_upgrade.go [hunk 7]
```diff
@@ -969,6 +918,184 @@ var _ = Describe("ROSA HCP cluster upgrade",
...
+				profile = profilesMap["rosa-hcp-upgrade-y-stream"]
+				profile.NamePrefix = constants.DefaultNamePrefix
+
+				flags, err := profilehandler.GenerateClusterCreateFlags(profile, rosaClient)
+				Expect(err).ToNot(HaveOccurred())
+
+				clusterName = common.GenerateRandomName("cluster-55883", 2)
+				command := "rosa create cluster --cluster-name " + clusterName + " " + strings.Join(flags, " ")
+				rosalCommand := config.GenerateCommand(command)
+				// rolePrefix = rosalCommand.GetFlagValue("--operator-roles-prefix", true)
```
Comment (radtriste) [inline tests/e2e/test_rosacli_upgrade.go:1041]: to be removed

File: tests/utils/exec/rosacli/cluster_service.go [hunk 8]
```diff
@@ -210,7 +210,7 @@ func (c *clusterService) ReflectClusterDescription(result bytes.Buffer) (res *Cl
 }
 
 func (c *clusterService) List() (bytes.Buffer, error) {
-	list := c.client.Runner.Cmd("list", "cluster")
+	list := c.client.Runner.Cmd("list", "cluster").CmdFlags()
```
Comment (radtriste) [inline tests/utils/exec/rosacli/cluster_service.go:213]: what is the purpose of calling `.CmdFlags()` with empty params ?

**Comment by aaraj7** [comment]
Sorry the PR got git issue, so closed it

**Pull request closed without merging**